### ISSUE TITLE
{Linked:Bug1465180; Linked:Bug1332753; Linked:Bug1515098; Linked:Bug1475892; Linked:Bug9919196; Linked:Bug9895710; Linked:Bug9895674; Linked:Bug9895727; Linked:Bug9895733; Linked:Bug9475345; Linked:Bug9651513; Linked:Bug9781446} [Accessibility] Ensure elements with Aria role that require child roles contain them

### ISSUE THREAD

**Issue by Raisul123** [microsoft/AdaptiveCards#8291]
### Target Platforms

NodeJS

### SDK Version

2.9.0, 2.11.1

### Application Name

Viva Connections

### Problem Description

https://onedrive.visualstudio.com/CSI/_workitems/edit/1465180

Elements with an ARIA role do not contain required child roles.
Required ARIA children role not present: group, menuitemradio, menuitem, menuitemcheckbox

Similar GH issue: https://github.com/microsoft/AdaptiveCards/issues/7604

### Screenshots

![Screenshot_1](https://user-images.githubusercontent.com/23114460/217191420-9af711be-5c15-4a85-9870-bede41e5d173.jpg)

### Card JSON

```JSON
{
    "schema": "http=//adaptivecards.io/schemas/adaptive-card.json",
    "type": "AdaptiveCard",
    "version": "1.3",
    "body": [
        {
            "id": "",
            "isVisible": true,
            "separator": false,
            "spacing": "default",
            "type": "Container",
            "items": [
                {
                    "id": "",
                    "isVisible": true,
                    "separator": false,
                    "spacing": "default",
                    "type": "TextBlock",
                    "textStyle": "default",
                    "wrap": true,
                    "text": "Phishing Validation",
                    "size": "Medium",
                    "weight": "Bolder"
                },
                {
                    "id": "",
                    "isVisible": true,
                    "separator": false,
                    "spacing": "default",
                    "type": "TextBlock",
                    "textStyle": "default",
                    "wrap": true,
                    "text": "Does the phishing email you are reporting look like the message below?",
                    "size": "Small",
                    "weight": "Bolder"
                },
                {
                    "id": "",
                    "isVisible": true,
                    "separator": false,
                    "spacing": "default",
                    "type": "Image",
                    "height": "auto",
                    "horizontalAlignment": "Left",
                    "size": "auto",
                    "style": "default",
                    "url": "https://th.bing.com/th/id/OIP.b5eA3HeWhZKHqLtSQos27gHaHP?w=170&h=180&c=7&r=0&o=5&dpr=1.5&pid=1.7",
                    "altText": "Phishing Email Image"
                }
            ],
            "verticalContentAlignment": "Center",
            "bleed": false
        },
        {
            "id": "",
            "isVisible": true,
            "separator": false,
            "spacing": "default",
            "type": "ColumnSet",
            "bleed": false,
            "columns": [
                {
                    "type": "Column",
                    "separator": false,
                    "spacing": "default",
                    "verticalContentAlignment": "Center",
                    "isVisible": true,
                    "items": [
                        {
                            "id": "",
                            "isVisible": true,
                            "separator": false,
                            "spacing": "default",
                            "type": "ActionSet",
                            "actions": [
                                {
                                    "id": "PhishingEmailSubmitNo",
                                    "title": "No",
                                    "altText": "No",
                                    "type": "Action.OpenUrl",
                                    "url": "https://aka.ms/dev-reportitnow-phishing"
                                }
                            ]
                        }
                    ],
                    "width": "stretch"
                },
                {
                    "type": "Column",
                    "separator": false,
                    "spacing": "default",
                    "verticalContentAlignment": "Center",
                    "isVisible": true,
                    "items": [
                        {
                            "id": "",
                            "isVisible": true,
                            "separator": false,
                            "spacing": "default",
                            "type": "ActionSet",
                            "actions": [
                                {
                                    "style": "positive",
                                    "id": "PhishingEmailSubmitYes",
                                    "title": "Yes",
                                    "altText": "Yes",
                                    "type": "Action.Submit",
                                    "associatedInputs": "auto",
                                    "data": {
                                        "id": "PhishingEmailSubmitYes"
                                    }
                                }
                            ]
                        }
                    ],
                    "width": "stretch"
                }
            ]
        }
    ],
    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json"
}
```

**Comment by Raisul123** [comment]
Copied from : https://github.com/microsoft/AdaptiveCards/issues/8258

Dependent Bug ID:
[Bug 1465180](https://onedrive.visualstudio.com/ODSP%20Product%20Experiences/_workitems/edit/1465180): [Screen Readers - My Hub Viva UX]: Ensures elements with an ARIA role that require child roles contain them.
[Bug 1332753](https://onedrive.visualstudio.com/WEX!/_workitems/edit/1332753): [Accessibility][Screen Readers-MyHub Viva UX-Dev Feedback][Sharepoint][Action.Submit]: Ensures elements with an ARIA role that require child roles contain them
[Bug 1515098](https://onedrive.visualstudio.com/ODSP%20Product%20Experiences/_workitems/edit/1515098): [Accessibility][Viva Connections – Dashboard – Top News card]: Ensures elements with an ARIA role that require child roles contain them
[Bug 1475892](https://onedrive.visualstudio.com/ODSP%20Product%20Experiences/_workitems/edit/1475892): [Screen Readers-MyHub Viva UX (Report It Now Feature)-Security Concern?]: Ensures elements with an ARIA role that require child roles contain them.
[Bug 9919196](https://microsoftit.visualstudio.com/OneITVSO/_workitems/edit/9919196): [Screen Reader- MyHub Viva UX-Account News Tile]: The text present below 'Account News' is exposed as a link in UI,but is programmatically declared as Paragraph.

**Comment by dennisjsyi** [comment]
Unable to reproduce with adaptive cards version 2.11.1

**Comment by jwoo-msft** [comment]
#8138

**Comment by jwoo-msft** [comment]
> Unable to reproduce with adaptive cards version 2.11.1

@dennisjsyi 
can we close this issue?

**Comment by dennisjsyi** [comment]
@jwoo-msft Yes we can close the issue.

**Comment by Raisul123** [comment]
Issue has been fixed on below test environment.
url: https://adaptivecards.io/designer/
Adaptive card version: 1.3 and 1.5
Edge Browser version: Version 111.0.1660.13 (Official build) dev (64-bit)
![8291-Code snipped-Not repro](https://user-images.githubusercontent.com/23114460/220894313-278fe6da-e73a-4323-825b-d55fdb8dd27b.jpg)
![8291-AI4W-Not repro](https://user-images.githubusercontent.com/23114460/220894348-60c352f5-c406-47e4-a1c2-0fce4a552b71.jpg)

Hence closing it.
